web/assets/main.scss

```diff
@@ -5,6 +5,9 @@ $moj-page-width: 1220px;
 @import "node_modules/govuk-frontend/govuk/all";
 @import "node_modules/@ministryofjustice/frontend/moj/all";
 @import "node_modules/accessible-autocomplete/src/autocomplete";
+@import "node_modules/tinymce/skins/content/default/content";
+@import "node_modules/tinymce/skins/ui/oxide/content";
+@import "node_modules/tinymce/skins/ui/oxide/skin";
```

Comment:
I don't think there's any point adding this here because TinyMCE loads in its own CSS files (and it does so inside an iframe, so these classes won't cascade through)